packages/dapi-grpc/scripts/build.sh

```diff
@@ -62,13 +59,13 @@ pbjs \
 rm -rf "$PLATFORM_WEB_OUT_PATH/*"
 
 docker run -v "$PLATFORM_PROTO_PATH:$PLATFORM_PROTO_PATH" \
-           -v "$PLATFORM_WEB_OUT_PATH:$PLATFORM_WEB_OUT_PATH" \
-           --rm \
-           "$PROTOC_IMAGE" \
-           --js_out="import_style=commonjs:$PLATFORM_WEB_OUT_PATH" \
-           --ts_out="service=grpc-web:$PLATFORM_WEB_OUT_PATH" \
-           -I="$PLATFORM_PROTO_PATH" \
-           "platform.proto"
+  -v "$PLATFORM_WEB_OUT_PATH:$PLATFORM_WEB_OUT_PATH" \
+  --rm \
+  "$PROTOC_IMAGE" \
+  --js_out="import_style=commonjs:$PLATFORM_WEB_OUT_PATH" \
+  --ts_out="service=grpc-web:$PLATFORM_WEB_OUT_PATH" \
+  -I="$PLATFORM_PROTO_PATH" \
+  "platform.proto"
```

Comment:
@markin-io To make sure that patch won't be missed shouldn't we just move it here instead of having additional call in package.json (`... && yarn exec scripts/patch-protobuf-js.sh`)

Reply:
@lklimek added it to the bottom of the script, so all good